src/ledger/LedgerManagerImpl.cpp

```diff
@@ -903,23 +904,23 @@ LedgerManagerImpl::processFeesSeqNums(std::vector<TransactionFramePtr>& txs,
     int index = 0;
     try
     {
-        soci::transaction sqlTx(mApp.getDatabase().getSession());
-        for (auto tx : txs)
-        {
-            LedgerDelta thisTxDelta(delta);
-            tx->processFeeSeqNum(thisTxDelta, *this);
-            tx->storeTransactionFee(*this, thisTxDelta.getChanges(), ++index);
-            thisTxDelta.commit();
-        }
-        sqlTx.commit();
-    }
-    catch (std::exception& e)
-    {
-        CLOG(FATAL, "Ledger")
-            << "processFeesSeqNums error @ " << index << " : " << e.what();
-        throw;
-    }
-}
+		soci::transaction sqlTx(mApp.getDatabase().getSession());
```

Comment:
I think this was caused by tab vs spacing setting differences in the different environments we're using to edit code.  This will be cleaned up by clang-format when we run it later.